notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -23,8 +23,18 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    fig, _ = make_plot(df_cyclic, adv_name="Cyclic")
</pre><p>should we highlight the s545.0mil point?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Ah I chose not to highlight it because it made the compute curve too difficult to read on the RHS. And there is a lot of action there.